spec/lib/contextual_logger_spec.rb

```diff
@@ -100,7 +101,13 @@ def expect_log_line_to_be_written(log_line)
       let(:log_level) { Logger::Severity::ERROR }
       let(:console_log_stream) { StringIO.new }
       let(:console_logger) { Logger.new(console_log_stream) }
-      let(:broadcast_logger) { logger.extend(ActiveSupport::Logger.broadcast(console_logger)) }
+      let(:broadcast_logger) do
+        if ::ActiveSupport::VERSION::STRING < "7.1"
```

Comment:
❓ Since you've opted to prevent this gem from being used with Rails 7.1, do we need this?

Reply:
We'll need to eventually, so I don't think it hurts to leave it 🤷

Reply:
I think YAGNI claims this one, if we opt to remove the broadcast feature, then we won't need it, and it's better to be able to see the tests breaking on 7.1 immediately than to have to remember to remove this as well.

Reply:
We'll see the problem immediately still. This is just making the interface change so that we're actually setting up the broadcast logger properly.

Reply:
Two changes were made in Rails 7.1:
1. How to setup the Broadcast logger
2. How the log lines are broadcast to the multiples loggers.

This change fixes number 1, but the bug exists with number 2.